### PR TITLE
Feature/activity-composer-vue: Create a new folder when selecting destination folder

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: ecm-wcm-extension/src/main/webapp/attachments-selector/components/modal/main.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import ExoModal from './ExoModal.vue';
+
+Vue.component('exo-news-modal', ExoModal);
```
Comment (thomasdelhomenie) [inline ecm-wcm-extension/src/main/webapp/attachments-selector/components/modal/main.js:3]: Its name should not contain "news".
Actually, we should move this component in a common location (https://github.com/Meeds-io/social/tree/develop/webapp/vue-portlet/src/main/webapp/common ?) to make it reusable everywhere.

File: ecm-wcm-extension/src/main/webapp/attachments-selector/attachmentsService.js [hunk 2]
```diff
@@ -29,4 +29,13 @@ export function getDrivers() {
...
     });
+}
+export function createFolder(currentDrive, workspace, parentPath, newFolderName) {
+  return fetch(`/portal/rest/managedocument/createFolder?driveName=${currentDrive}&workspaceName=${workspace}&currentFolder=${parentPath}&folderName=${newFolderName}`,
+    {})
+    .then(response => response.text())
+    .then(xmlStr => (new window.DOMParser()).parseFromString(xmlStr, 'text/xml'))
+    .then(xml => {
+      return xml;
+    });
```
Comment (thomasdelhomenie) [inline ecm-wcm-extension/src/main/webapp/attachments-selector/attachmentsService.js:40]: There should be catch block to handle errors.

File: ecm-wcm-extension/src/main/webapp/attachments-selector/components/modal/main.js (hunk 1 above)
Comment (aelbenney) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/components/modal/main.js:3]: Yesss, good idea!

File: ecm-wcm-extension/src/main/webapp/attachments-selector/attachmentsService.js (hunk 2 above)
Comment (aelbenney) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/attachmentsService.js:40]: okay

**Pull request merged**